src/test/java/com/gooddata/collections/Elements.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2004-2017, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.collections;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+import java.util.List;
+import java.util.Map;
+
+@JsonDeserialize(using = ElementsDeserializer.class)
+@JsonTypeInfo(include = JsonTypeInfo.As.WRAPPER_OBJECT, use = JsonTypeInfo.Id.NAME)
+@JsonTypeName(Elements.ROOT_NODE)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonSerialize(using = ElementsSerializer.class)
+public class Elements extends PageableList<String> {
```

Comment:
does it have to be public?

Reply:
it is in test sources, so i guess it doesn't matter